packages/payer-logo/PayerLogo.js

```diff
@@ -27,18 +28,66 @@ query($payerIDs: [String!], $types: [String!]){
 }
 `;
 
-export default class PayerLogo extends Component {
-  state = {};
+const getLogoBySpaceId = async spaceId =>
+  new Promise(async (resolve, reject) => {
```

Comment:
I think we can do away with creating a promise here and just use `async`/`await` syntax. Will reduce all the extra setup for the `resolve/reject`.

Reply:
please see 337253dfc06c9b6fbdedcdbd31b782e696f64f55